mindee/client.py

```diff
@@ -194,7 +202,10 @@ def _make_request(
     ) -> PredictResponse[TypeDocument]:
         if get_bound_classname(document_class) != doc_config.document_class.__name__:
             raise RuntimeError("Document class mismatch!")
-
+        if self.input_doc is None:
+            raise RuntimeError(
+                "The '_make_request' class method requires an input document."
```

Comment:
Maybe use `TypeError` instead?

Because `None` is of type `NoneType`, which is of a different type than `XxxInputSource`...

https://docs.python.org/3/library/exceptions.html#TypeError